search_commands.go

```diff
@@ -0,0 +1,1215 @@
+		}
+		if schema.Weight > 0 {
+			args = append(args, "WEIGHT", schema.Weight)
+		}
+		if schema.Seperator != "" {
+			args = append(args, "SEPERATOR", schema.Seperator)
+		}
+		if schema.CaseSensitive {
+			args = append(args, "CASESENSITIVE")
+		}
+		if schema.WithSuffixtrie {
+			args = append(args, "WITHSUFFIXTRIE")
+		}
+	}
+	cmd := NewStatusCmd(ctx, args...)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+// FTCursorDel - Deletes a cursor from an existing index.
+// The 'index' parameter specifies the index from which to delete the cursor, and the 'cursorId' parameter specifies the ID of the cursor to delete.
+// For more information, please refer to the Redis documentation: [FT.CURSOR DEL](https://redis.io/commands/ft.cursor-del/)
+func (c cmdable) FTCursorDel(ctx context.Context, index string, cursorId int) *StatusCmd {
+	cmd := NewStatusCmd(ctx, "FT.CURSOR", "DEL", index, cursorId)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+// FTCursorRead - Reads the next results from an existing cursor.
+// The 'index' parameter specifies the index from which to read the cursor, the 'cursorId' parameter specifies the ID of the cursor to read, and the 'count' parameter specifies the number of results to read.
+// For more information, please refer to the Redis documentation: [FT.CURSOR READ](https://redis.io/commands/ft.cursor-read/)
+func (c cmdable) FTCursorRead(ctx context.Context, index string, cursorId int, count int) *MapStringInterfaceCmd {
+	args := []interface{}{"FT.CURSOR", "READ", index, cursorId}
+	if count > 0 {
+		args = append(args, "COUNT", count)
+	}
+	cmd := NewMapStringInterfaceCmd(ctx, args...)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+// FTDictAdd - Adds terms to a dictionary.
+// The 'dict' parameter specifies the dictionary to which to add the terms, and the 'term' parameter specifies the terms to add.
+// For more information, please refer to the Redis documentation: [FT.DICTADD](https://redis.io/commands/ft.dictadd/)
+func (c cmdable) FTDictAdd(ctx context.Context, dict string, term []interface{}) *IntCmd {
+	args := []interface{}{"FT.DICTADD", dict}
+	args = append(args, term...)
+	cmd := NewIntCmd(ctx, args...)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+// FTDictDel - Deletes terms from a dictionary.
+// The 'dict' parameter specifies the dictionary from which to delete the terms, and the 'term' parameter specifies the terms to delete.
+// For more information, please refer to the Redis documentation: [FT.DICTDEL](https://redis.io/commands/ft.dictdel/)
+func (c cmdable) FTDictDel(ctx context.Context, dict string, term []interface{}) *IntCmd {
+	args := []interface{}{"FT.DICTDEL", dict}
+	args = append(args, term...)
+	cmd := NewIntCmd(ctx, args...)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+// FTDictDump - Returns all terms in the specified dictionary.
+// The 'dict' parameter specifies the dictionary from which to return the terms.
+// For more information, please refer to the Redis documentation: [FT.DICTDUMP](https://redis.io/commands/ft.dictdump/)
```

Comment:
Does the markdown render on go docs?

Reply:
fixed to match:
https://tip.golang.org/doc/comment#links